### ISSUE TITLE
Port for Windows?

### ISSUE THREAD

**Issue by arthurrasmusson** [rviscarra/webrtc-remote-screen#7]
I've had success running this on Linux but I'd like to build it on my Windows machine also.

Has anyone had any success with getting this wonderful project working on Windows?

I will try my best to figure it out and post back here if I manage to get it working.

**Comment by naikrovek** [comment]
I got it working once, and I forgot what I did.  Trying to reproduce my success now.  Will update if I succeed.

**Comment by panbak** [comment]
@naikrovek Were you able to reproduce?

**Comment by naikrovek** [comment]
no.  I can't build it anymore, and I can't remember how I got around this previously.  :(
